### ISSUE TITLE
Maximise data by choosing best event lists and symlinking

### ISSUE THREAD

**Issue by DavidT3** [DavidT3/XGA#421]
Currently any analysis where there is an ObsID with an exp2 isn't fully exploiting all available data, especially as many exp1s can be cal closed observations which obviously XGA won't touch.

I don't want to rework XGA to support an arbitrary number of datasets per ObsID-Instrument, it would be a breaking change and require a significant chunk of work.

On apollo, for those observations with exp2s, I should determine which exps are valid, and then select the best and make a 'best_pn_clean_evts' symlink (for instance). 

This is a stopgap, and perhaps someday I shall implement a way in XGA to use all the data, but this would provide a significant improvement for minimal effort

**Comment by DavidT3** [comment]
This is almost a duplicate of issue #128, which shows you that I should have fixed this long ago. However I don't believe changing things on the XGA end is the answer, as I mentioned above.

**Comment by DavidT3** [comment]
So I think I've decided on the best way to do this, with Paul's help. I'll make a separate xga_data directory in the xmm_obs directory, with all ObsIDs in there as normal.

Then I'll just symlink the files I need there, including choosing the 'best' files from observations with multiple sub-exposures:

This was part of the email I sent to XCS explaining the criteria I used:

"Basically I've gone through all the ObsIDs with multiple sub-exposures (so anything with an exp2 or greater events list) and selected the 'best' sub-exposure for each instrument of an observation by finding the events list with the longest LIVETIME that doesn't have the filter wheel closed for calibration (so FILTER != CalClosed).

What I want to do now is create symlinks from those 'best' event lists (and their corresponding images/exposure maps) to a predictable name that I can point XGA to, so "0201903501_pn_best_evts.fits" for instance, of course I would also have to do this for all the observations with only one sub-exposure, but in that case I'd obviously just point the symlinks at the exp1 event list etc."
